data/mods/Magiclysm/worldgen/orc_village.json

```
@@ -0,0 +1,64 @@
+[
+  {
+    "type": "palette",
+    "id": "orc_palette",
+    "terrain": { "3": "t_door_makeshift_c", "1": "t_wall_rammed_earth", ".": "t_dirtfloor" }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "orc_village",
+    "object": {
+      "fill_ter": "t_dirt",
+      "rows": [
+        "11111111111111111111111",
+        "1.....................1",
+        "1.....................3",
+        "1.....................1",
+        "1.....................3",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................3",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "1.....................1",
+        "11111111111111111111111"
+      ],
```

```suggestion
      ],
      "palettes": [ "orc_palette" ]
```
The only reason I can see for the function in question to not give you anything other than "format: no terrain map" is for not having a palette available.

---

I don't understand the reason either, hopefully it's fixed.

---

This error was indeed fixed, thank you.